jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/CodeWhispererFileContextProvider.kt

```diff
@@ -240,7 +240,7 @@ class DefaultCodeWhispererFileContextProvider(private val project: Project) : Fi
                     },
                     path = relativePath
                 )
-            )
+            ).filter { chunk -> chunk.content.isNotBlank() }
```

Comment:
if you're always appended and prepending to the content, won't it never be blank?

Reply:
it's an edge case when the opened file has empty content lol